stdlib/REPL/src/docview.jl

```diff
@@ -240,7 +240,8 @@ repl_corrections(s) = repl_corrections(stdout, s)
 const symbols_latex = Dict{String,String}()
 function symbol_latex(s::String)
     if isempty(symbols_latex) && isassigned(Base.REPL_MODULE_REF)
-        for (k,v) in Base.REPL_MODULE_REF[].REPLCompletions.latex_symbols
```

Comment:
Couldn't the whole `Base.REPL_MODULE_REF[].` be removed?

Reply:
Probably?  I guess that is an artifact of the old days when REPLCompletions was not part of the REPL module?

Reply:
Hm, right. Removed now.